class SectionNotFoundError(Exception):
    pass


class UserAlreadyExistsError(Exception):
    pass
